test/ClientHttpBinIntegrationTest.php

```diff
@@ -21,17 +21,18 @@
 use Amp\Http\Rfc7230;
 use Amp\Http\Server\Options;
 use Amp\Http\Server\Request as ServerRequest;
-use Amp\Http\Server\RequestHandler\CallableRequestHandler;
+use Amp\Http\Server\RequestHandler\ClosureRequestHandler;
 use Amp\Http\Server\Response as ServerResponse;
 use Amp\Http\Server\Server;
 use Amp\PHPUnit\AsyncTestCase;
+use Amp\Pipeline\Pipeline;
 use Amp\Socket;
+use Closure;
 use Psr\Log\NullLogger;
 use Revolt\EventLoop;
+use function _HumbugBoxae3c412aa099\Amp\asyncCall;
```

Comment:
Oops?

Reply:
Gah PHPStorm tried to autocomplete that...